boards/ark/can-flow/canbootloader.px4board

```diff
@@ -0,0 +1,5 @@
+CONFIG_BOARD_TOOLCHAIN="arm-none-eabi"
+CONFIG_BOARD_ARCHITECTURE="cortex-m4"
+CONFIG_BOARD_ROMFSROOT=""
```

Comment:
Instead of `CONFIG_BOARD_ROMFSROOT=""` can it be skipped entirely?

Reply:
Default is "px4fmu_common" as set in
https://github.com/PX4/PX4-Autopilot/blob/fc6f41c6878659354fc13139699a92783e1caddd/Kconfig#L60-L64

Thus setting it empty causes this, we could however just don't give default for CONFIG_BOARD_ROMFSROOT then this behaviour is gone.

Reply:
Can the default be skipping ROMFS entirely unless you specify?

Reply:
Yes that's what I mean, I remove the `default "px4fmu_common"` line and thus you've explicitly specify the romfsroot. I can change that tomorrow.